config.js

```diff
@@ -0,0 +1,61 @@
+const envSchema = require('env-schema')
```

Comment:
The other lambdas load their config from the `utils.js` file, which is done for 2 reasons:

1) Allows us to load from `.env` in local development environment, and from SSM/secrets manager in prod
2) Only loads config that the lambda actually uses, allowing us to lock down access to specific params/secrets for each lambda

Reply:
I found while tinkering with this code that the .env in fact was not being read.  I figured it was because we had yet to implement a similar mechanism as found in the `backend` repo.  Pls advise?  Secrets manager makes total sense, as does containing this functionality within the utils file.

Reply:
Done